pkg/opensearch/opensearch_test.go

```diff
@@ -30,3 +35,93 @@ func Test_wrapError(t *testing.T) {
 		assert.Equal(t, "OpenSearch data source error: some error", err.Error())
 	})
 }
+
+func TestServiceMapPreFetch(t *testing.T) {
+	targets1 := map[string]interface{}{
```

Comment:
this is definitely a "sarah doesn't understand go" question, but is there a way to rewrite this so we're looking at a json object or struct something so it's easier to read? Or is that like "not done" really?

Reply:
We can totally write these in JSON and then decode them - that will be more readable indeed.

Reply:
Done!